### PR TITLE
Cache refactoring + Test Cases refactoring

### PR DESCRIPTION
**Pull request by r0goyal**

### CODE REVIEW

**Comment by r0goyal** [comment]
@santanusinha Please review this

**Comment by r0goyal** [comment]
@santanusinha Is this reviewed ?

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/cache/CacheManager.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by rishabh.goyal on 26/12/15.
+ */
+
+public class CacheManager {
+
+    private final Map<String, Cache> cacheMap = new HashMap<>();
```
Comment (santanusinha) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/cache/CacheManager.java]: Guava... :P

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/cache/CacheManager.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.flipkart.foxtrot.core.cache;
```
Comment (santanusinha) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/cache/CacheManager.java]: License

**Pull request merged**